### DISCUSSION TITLE
Dark mode without js button

### DISCUSSION THREAD

**Discussion by otkaudes** [twbs/bootstrap#37972] [Q&A]
Thanks for creating a dark mode. Is it possible to implement without the js toggle button?  I see mention of it on getbootstrap.com but is requires experience with SASS. I think the js button is redundant when "prefers-color-scheme" already knows the user's preference.

**Comment by andrerom** [comment]
You can do this with `$color-mode-type: media-query;` as explained here, but yes that is a sass variable:
https://getbootstrap.com/docs/5.3/customize/color-modes/#building-with-sass

If you prefer to use the pre built versions from CDN you'll have to handle it in Javascript similar to the code here:
https://getbootstrap.com/docs/5.3/customize/color-modes/#javascript

Might be they simplify this towards the stable release, IMO could argue standard should be css using media query to set CSS variables so defaults work with prefers-color-scheme while still allowing global and local overrides.

**Comment by o-alquimista** [comment]
Yeah, we have `prefers-color-scheme` in CSS to do the switching for us. I wish we had a way to just use that, without JavaScript or Sass. Perhaps by adding a `data-bs-theme="auto"`.
